### DISCUSSION TITLE
Cursed or blessed magical items

### DISCUSSION THREAD

**Discussion by Dwfreegethub** [littlesera/LSCG#327] [Ideas]
Would it be possible to cast spells directly onto an object.. You could craft the item with the specific spell name in the description and whoever wore it would either be affected by it right away or have to make savings throws each round, or x rounds, against the effect.

And potentially the savings throw gets harder each round.  Or blessed items you have to roll to keep the effects.

**Comment by littlesera** [comment]
Interesting idea for #299

**Comment by littlesera** [comment]
Incl. #338
